app/src/main/java/practicum/getfitla_v3/ExerciseListAdapter.java

```diff
@@ -39,25 +36,28 @@ public itemViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     public void onBindViewHolder(itemViewHolder holder, int position) {
         ExerciseItemFormat exercise = fullList.get(position);
 
-        holder.textViewTitle.setText(exercise.getTitle());
+        holder.textViewTitle.setText(exercise.getName());
         holder.textViewShortDesc.setText(exercise.getShortdesc());
-        holder.textViewRating.setText(String.valueOf(exercise.getRating()));
-        holder.textViewPrice.setText(String.valueOf(exercise.getPrice()));
+        //holder.textViewRating.setText(String.valueOf(exercise.getRating()));
+        //holder.textViewPrice.setText(String.valueOf(exercise.getPrice()));
         holder.imageview.setImageDrawable(mtcx.getResources().getDrawable(exercise.getImage()));
 
+
     }
 
     @Override
     public int getItemCount() {
         return fullList == null ? 0 : fullList.size();
```

Comment:
This is probably fine, but I usually just write 
```
return fullList.size();
``` 
I don't think you need to catch the exception when it's 0